src/OrleansSQLUtils/Storage/RelationalStorage.cs

```diff
@@ -286,7 +264,7 @@ private bool IsSynchronousAdoNetImplementation
                     command.CommandText = query;
 
                     Task<Tuple<IEnumerable<TResult>, int>> ret;
-                    if(IsSynchronousAdoNetImplementation)
+                    if(isSynchronousAdoNetImplementation)
```

Comment:
@shayhatsor I left an unfortunate bug here the last time. Ultimately this capability is detected with this piece of code:

```
public static bool IsSynchronousAdoNetImplementation(string adoNetProvider)
        {
            //Currently the assumption is all but MySQL support DbCommand cancellation.            
            return !adoNetProvider.Equals(AdoNetInvariants.InvariantNameMySql, StringComparison.OrdinalIgnoreCase);
        }
```
As you can see, it shouldn't be negated.